src/apdu_pubkey.c

```diff
@@ -49,53 +50,42 @@ static void bip32_path_to_string(char *const out, size_t const out_size, apdu_pu
     }
 }
 
-static void render_pkh(const char *const hrb, char *const out, size_t const out_size,
-                       apdu_pubkey_state_t const *const pubkey) {
+void render_pkh(char *const out, size_t const out_size,
+                       prefixed_public_key_hash_t const *const pubkey) {
     const size_t base32_max = 256;
     uint8_t base32_buf[base32_max];
     size_t base32_len = 0;
     if (!convert_bits(base32_buf, base32_max, &base32_len,
                       5,
-                      pubkey->prefixed_public_key_hash.entire, sizeof(pubkey->prefixed_public_key_hash.entire),
+                      pubkey->entire, sizeof(pubkey->entire),
                       8,
                       1)) {
         THROW(EXC_MEMORY_ERROR);
     }
-    if (!bech32_encode(out, out_size, hrb, base32_buf, base32_len)) {
+    if (!bech32_encode(out, out_size, N_data.show_testnet?"ckt":"ckb", base32_buf, base32_len)) {
```

Comment:
An enum might be better here instead of show_testnet. At least for tezos we had up to 3 or 4 networks at one time.

Although they might always just use ckt addresses?

Reply:
So far they're all ckt.